src/UI/Pages/ResourceDetails/Tabs/FactsTab/FactsTable.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useEffect, useState } from "react";
+import {
+  TableComposable,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from "@patternfly/react-table";
+import { Fact } from "@/Core";
+import { Order } from "@/Core/Domain/Sort";
+import { ColumnHead } from "@/UI/Presenters";
+import { words } from "@/UI/words";
+
+type FactRow = Pick<Fact, "id" | "name" | "updated" | "value">;
+
+interface Props {
+  facts: FactRow[];
+}
+
+const factsColumnHeads: ColumnHead[] = [
+  { apiName: "name", displayName: words("resources.facts.columns.name") },
+  { apiName: "updated", displayName: words("resources.facts.columns.updated") },
+  { apiName: "value", displayName: words("resources.facts.columns.value") },
+];
+
+export const FactsTable: React.FC<Props> = ({ facts }) => {
+  const [activeSortColumn, setActiveSortColumn] = useState("name");
```

Comment:
We no longer have this sorting as 2 separate properties. We have a `Sort` type that covers this.
I think it is better to use that. Not sure if this sorting needs to be saved in the url.

Reply:
The `Sort` type is a good idea.
Saving it in the url is not necessary I think since the endpoint is not paged